backend/src/main/java/gov/cdc/usds/simplereport/api/model/filerow/FileRow.java

```diff
@@ -0,0 +1,10 @@
+package gov.cdc.usds.simplereport.api.model.filerow;
+
+import gov.cdc.usds.simplereport.service.model.reportstream.FeedbackMessage;
+import java.util.List;
+
+public interface FileRow {
+  List<FeedbackMessage> validateHeaders();
```

Comment:
I'll change it! There are some items in that section that aren't even marked as required, [like comment](https://github.com/CDCgov/prime-simplereport/blob/f74cd4a9ddc93a5d081f84cc3cc511c8a70571ab/backend/src/main/java/gov/cdc/usds/simplereport/validators/TestResultFileValidator.java#L124-L125), and then [still being validated.](https://github.com/CDCgov/prime-simplereport/blob/f74cd4a9ddc93a5d081f84cc3cc511c8a70571ab/backend/src/main/java/gov/cdc/usds/simplereport/validators/TestResultFileValidator.java#L188-L189). Would it be OK to use the fields that are marked `required=true` only in that section?

Reply:
this is a good point. `field.getPossibleError()` is only going to return something if `field` was created with `required=true` so yeah checking the others is kind of pointless and I do think it would be OK to only use required fields for that section. I could see it being potentially easier to upkeep if all the fields are checked because someone might make a field required or create a new one and forget to add it to the validate required fields method but now that you've renamed the method that should make it pretty clear imo

Reply:
I took a stab at something you said about someone forgetting to validate the field if it is required. This honestly may have introduced way more complexity than needed though because of reflection, [2f3e8a5](https://github.com/CDCgov/prime-simplereport/pull/4730/commits/2f3e8a5c689f7cafcbe11e7fb2c58e1a40a7a394).